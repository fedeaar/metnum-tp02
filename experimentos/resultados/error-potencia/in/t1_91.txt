-2.684447914293302233e+02 3.822963659176816797e+00 1.665345326486158584e+01 9.013443871645463901e+00 -2.366906368171716224e+01 1.360445586377918570e+00 3.927667246976025694e+01 1.820638404228830254e+01 1.046960867214823168e+02 2.184289303453956776e+00 2.102678613604125779e+00 5.958002460087501362e+01 3.704863634135067940e+01 -8.686633135787246829e+00 1.187912080690941607e+01 -2.217817433885966416e+00 -5.533091535823489338e+00 -3.598691566396372998e+00 2.001394154148028015e+00 6.422403765224981953e+01
3.822963659176816797e+00 2.977792457252018608e+02 1.171930462057028954e+01 5.136012663076900964e+00 -1.291714275888363659e+02 -9.788666040343275698e-01 -5.870553351843098255e+01 2.008119188439345493e+01 8.547452062802643979e+01 -2.025343364096209076e+00 -6.179088230844219432e+00 5.833710859813184157e+01 1.676760788554715020e+01 -6.212204665889688471e+01 9.040901693907846592e+00 -1.210226338400245396e+02 -1.545565437308575838e+02 -1.118646888962847328e+02 3.076587199549735141e-02 4.837133868934992620e+01
1.665345326486158228e+01 1.171930462057028421e+01 -5.480718208246784116e+02 6.585131640359908722e+00 -4.910575012821846386e+00 1.181667750703247366e+00 3.737385549471913038e+01 1.226246463420992328e+01 7.372670192820365287e+01 1.945809652204102180e+00 2.325787495274289185e+00 4.091836222152537061e+01 2.753223259358939146e+01 -2.271687198766254578e-01 8.435594605210930297e+00 1.119758646092268251e+01 1.216245316683215805e+01 9.124333773797054903e+00 1.580966648446940903e+00 4.566110606831972518e+01
9.013443871645463901e+00 5.136012663076900964e+00 6.585131640359909611e+00 -4.703837626865305310e+02 -4.499439660900776516e+00 6.215801641573643899e-01 1.926030366547535877e+01 6.885224292190724960e+00 4.088947832766135093e+01 1.017586555318877428e+00 1.162246167975683919e+00 2.285044029503442786e+01 1.505169177068620456e+01 -1.015704098147739431e+00 4.667828268730752406e+00 4.282891715060653581e+00 4.319607558276249648e+00 3.299337538213658050e+00 8.508981620521625100e-01 2.525838547452174510e+01
-2.366906368171716579e+01 -1.291714275888363659e+02 -4.910575012821844609e+00 -4.499439660900780069e+00 7.022866517508426796e+02 -3.355751378346009783e+00 -1.433399623650579144e+02 5.723754011239581274e+00 -1.286809093046301555e+01 -6.080222942002176723e+00 -1.230693451213155143e+01 7.472115487533159239e+00 -2.512209747790240755e+01 -8.290578243917427415e+01 -2.463035173808192368e+00 -1.816391119883537044e+02 -2.282725945455088947e+02 -1.657745244491077301e+02 -2.692370816846674231e+00 -1.409144795209153500e+01
1.360445586377918348e+00 -9.788666040343276808e-01 1.181667750703247366e+00 6.215801641573643899e-01 -3.355751378346009339e+00 -5.993231434571053740e+01 1.500533767469217405e+00 1.400154558895633006e+00 7.604621704903283153e+00 1.019145598761416333e-01 3.509573461973351793e-02 4.472055268814551177e+00 2.490219542122693142e+00 -1.450811056441807745e+00 8.530499632053299974e-01 -1.937145193806332877e+00 -2.637224528596994411e+00 -1.884212013947290876e+00 1.214869846711232604e-01 4.604415943642333531e+00
3.927667246976024984e+01 -5.870553351843098966e+01 3.737385549471913038e+01 1.926030366547536232e+01 -1.433399623650579144e+02 1.500533767469216961e+00 6.590821910923865801e+01 4.668774574399493105e+01 2.444205820162546274e+02 2.045401393635297715e+00 -1.422428699191631285e+00 1.468684188789517862e+02 7.568432270906454562e+01 -6.440621922114559084e+01 2.720561460890608529e+01 -1.007692203752276328e+02 -1.332282070893947434e+02 -9.574558069456529097e+01 3.386866654983170299e+00 1.466788512199013326e+02
1.820638404228829899e+01 2.008119188439345493e+01 1.226246463420992150e+01 6.885224292190724960e+00 5.723754011239581274e+00 1.400154558895632562e+00 4.668774574399493105e+01 -9.380898056586959228e+02 7.466334947158703983e+01 2.341402955417349219e+00 3.124201601254951388e+00 4.049406701086103055e+01 2.919457478078694379e+01 5.128653413230120250e+00 8.606767851518716483e+00 2.294841503947207784e+01 2.692742639194962706e+01 1.984726596590248704e+01 1.757165914033221865e+00 4.663670360980235330e+01
1.046960867214823168e+02 8.547452062802646822e+01 7.372670192820366708e+01 4.088947832766135093e+01 -1.286809093046298536e+01 7.604621704903279600e+00 2.444205820162546274e+02 7.466334947158702562e+01 -1.921370535583688195e+02 1.258039282022187066e+01 1.556551408687217375e+01 2.503618018655768651e+02 1.716193163283555805e+02 7.299095370667370020e+00 5.203356117864755248e+01 8.777405554683681999e+01 9.858606050628496575e+01 7.338547771382619089e+01 9.985844023782407319e+00 2.817325475770322214e+02
2.184289303453956776e+00 -2.025343364096208632e+00 1.945809652204102180e+00 1.017586555318877206e+00 -6.080222942002176723e+00 1.019145598761416888e-01 2.045401393635297715e+00 2.341402955417348775e+00 1.258039282022187066e+01 -2.584973532292727327e+01 2.005157852487876591e-02 7.444832218556827641e+00 4.054707816830612543e+00 -2.664988123065885173e+00 1.408044118838159164e+00 -3.778486267662450437e+00 -5.085026831674014147e+00 -3.641408175305923756e+00 1.932597543450685906e-01 7.597574864844392373e+00
2.102678613604125779e+00 -6.179088230844219432e+00 2.325787495274290073e+00 1.162246167975683475e+00 -1.230693451213155143e+01 3.509573461973351793e-02 -1.422428699191631285e+00 3.124201601254951388e+00 1.556551408687217020e+01 2.005157852487883530e-02 1.586809433639727445e+02 9.633621872528618013e+00 4.429795510452874119e+00 -5.693962586733797515e+00 1.713524594270561696e+00 -9.866849951361235327e+00 -1.282596814839430444e+01 -9.249311349276164052e+00 1.692969891322443077e-01 9.223470570069615349e+00
5.958002460087501362e+01 5.833710859813181315e+01 4.091836222152537061e+01 2.285044029503442431e+01 7.472115487533149469e+00 4.472055268814551177e+00 1.468684188789517577e+02 4.049406701086103055e+01 2.503618018655768651e+02 7.444832218556827641e+00 9.633621872528618013e+00 -6.631804099754616573e+02 9.645727571820238211e+01 1.132569049136118977e+01 2.879015722783435649e+01 6.423084252249097403e+01 7.428400866400019709e+01 5.492948914159129004e+01 5.721083577167312306e+00 1.559492356163019338e+02
3.704863634135067940e+01 1.676760788554714665e+01 2.753223259358939146e+01 1.505169177068620456e+01 -2.512209747790237557e+01 2.490219542122693142e+00 7.568432270906454562e+01 2.919457478078695090e+01 1.716193163283555805e+02 4.054707816830612543e+00 4.429795510452874119e+00 9.645727571820238211e+01 -3.475911342010008980e+02 -7.387733947917897837e+00 1.955425898608165269e+01 1.120696901633071363e+01 9.610666516116687319e+00 7.662852983957932906e+00 3.480317710535960174e+00 1.057826396483023643e+02
-8.686633135787250382e+00 -6.212204665889689892e+01 -2.271687198766254578e-01 -1.015704098147741430e+00 -8.290578243917425993e+01 -1.450811056441807967e+00 -6.440621922114559084e+01 5.128653413230122027e+00 7.299095370667369131e+00 -2.664988123065884729e+00 -5.693962586733797515e+00 1.132569049136118977e+01 -7.387733947917897837e+00 6.816879421360927154e+02 3.419759434033640133e-01 -8.833701705342170385e+01 -1.113714341584480820e+02 -8.082514088792578377e+01 -1.046361669318117915e+00 1.473129831617519869e+00
1.187912080690941607e+01 9.040901693907851921e+00 8.435594605210930297e+00 4.667828268730753294e+00 -2.463035173808187484e+00 8.530499632053301085e-01 2.720561460890608529e+01 8.606767851518716483e+00 5.203356117864755959e+01 1.408044118838159164e+00 1.713524594270561696e+00 2.879015722783435649e+01 1.955425898608165269e+01 3.419759434033662338e-01 -5.960404718654617682e+02 8.990979090442671406e+00 9.953330543863554425e+00 7.433872708978206845e+00 1.130420893449099573e+00 3.226298151565001149e+01
-2.217817433885976186e+00 -1.210226338400245396e+02 1.119758646092268961e+01 4.282891715060651805e+00 -1.816391119883537044e+02 -1.937145193806332877e+00 -1.007692203752276328e+02 2.294841503947207784e+01 8.777405554683684841e+01 -3.778486267662451326e+00 -9.866849951361237103e+00 6.423084252249097403e+01 1.120696901633070830e+01 -8.833701705342170385e+01 8.990979090442669630e+00 2.895524706725971100e+02 -2.256327007665598217e+02 -1.634559241524926563e+02 -6.834328108906031485e-01 4.786121146332556719e+01
-5.533091535823491114e+00 -1.545565437308575554e+02 1.216245316683216515e+01 4.319607558276246095e+00 -2.282725945455088095e+02 -2.637224528596993522e+00 -1.332282070893947150e+02 2.692742639194962706e+01 9.858606050628500839e+01 -5.085026831674013259e+00 -1.282596814839430444e+01 7.428400866400019709e+01 9.610666516116694424e+00 -1.113714341584480820e+02 9.953330543863554425e+00 -2.256327007665597648e+02 2.084386720315793866e+02 -2.076461689956388454e+02 -1.121670869837620765e+00 5.285978572075570270e+01
-3.598691566396373887e+00 -1.118646888962847044e+02 9.124333773797054903e+00 3.299337538213658050e+00 -1.657745244491077017e+02 -1.884212013947290876e+00 -9.574558069456529097e+01 1.984726596590248704e+01 7.338547771382621931e+01 -3.641408175305924200e+00 -9.249311349276164052e+00 5.492948914159128293e+01 7.662852983957918696e+00 -8.082514088792578377e+01 7.433872708978210397e+00 -1.634559241524926563e+02 -2.076461689956388454e+02 3.385445952741158067e+02 -7.744254818990224942e-01 3.950109238884658680e+01
2.001394154148027571e+00 3.076587199549735141e-02 1.580966648446941125e+00 8.508981620521625100e-01 -2.692370816846674231e+00 1.214869846711232604e-01 3.386866654983170299e+00 1.757165914033222087e+00 9.985844023782407319e+00 1.932597543450685629e-01 1.692969891322442522e-01 5.721083577167312306e+00 3.480317710535960174e+00 -1.046361669318117915e+00 1.130420893449099351e+00 -6.834328108906030375e-01 -1.121670869837620543e+00 -7.744254818990222722e-01 -2.338154545203524606e+02 6.109568946864204264e+00
6.422403765224981953e+01 4.837133868934991909e+01 4.566110606831971808e+01 2.525838547452174510e+01 -1.409144795209152079e+01 4.604415943642333531e+00 1.466788512199013326e+02 4.663670360980235330e+01 2.817325475770322214e+02 7.597574864844391485e+00 9.223470570069615349e+00 1.559492356163019338e+02 1.057826396483023643e+02 1.473129831617512764e+00 3.226298151565001149e+01 4.786121146332560272e+01 5.285978572075568849e+01 3.950109238884657259e+01 6.109568946864204264e+00 -4.213419856751227144e+02
