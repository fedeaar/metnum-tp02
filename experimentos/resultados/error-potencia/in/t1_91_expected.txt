-9.500000000000000000e+02
8.670000000000000000e+02
-8.000000000000000000e+02
7.230000000000000000e+02
-6.460000000000000000e+02
-6.020000000000000000e+02
-5.960000000000000000e+02
-5.600000000000000000e+02
4.950000000000000000e+02
4.890000000000000000e+02
-4.740000000000000000e+02
4.670000000000000000e+02
-4.100000000000000000e+02
3.790000000000000000e+02
-2.900000000000000000e+02
-2.340000000000000000e+02
1.590000000000000000e+02
-6.000000000000000000e+01
4.700000000000000000e+01
-2.600000000000000000e+01
